src/PrestaShopBundle/Twig/Extension/ColumnExtension.php

```diff
@@ -0,0 +1,88 @@
+<?php
+/**
+ * 2007-2018 PrestaShop
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to http://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2018 PrestaShop SA
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ */
+
+namespace PrestaShopBundle\Twig\Extension;
+
+use Error;
+use Symfony\Component\Templating\EngineInterface;
+use Twig\Extension\AbstractExtension;
+use Twig_SimpleFunction as SimpleFunction;
+
+/**
+ * Class ColumnExtension is responsible for providing twig functions:
+ *
+ * 1. column_content(column, row, grid) - renders column content based on column type
+ * 2. column_header(column, grid) - renders column header based on column type
+ * 3. column_flter(column, grid) - renders column filter based on column type
+ */
+class ColumnExtension extends AbstractExtension
+{
+    /**
+     * @var EngineInterface
+     */
+    private $templating;
+
+    public function __construct(EngineInterface $templating)
```

Comment:
rely on Twig Loader instead:

```php
if ($this->get('twig')->getLoader()->exists('@PrestaShop/Admin/Common/Grid/Columns/foo.html.twig')) {
    // ...
}
```

Templating component doesn't exists in Symfony 4 and I plan to disable it before 1.7.6 (but when exactly, I don't know).